src/cmd-buildupload

```diff
@@ -164,8 +160,19 @@ def s3_check_exists(bucket, key):
 
 @retry(stop=retry_stop, retry=retry_boto_exception, retry_error_callback=retry_callback)
 def s3_copy(src, bucket, key, max_age, acl, extra_args={}, dry_run=False):
-    if key.endswith('.json') and 'ContentType' not in extra_args:
-        extra_args['ContentType'] = 'application/json'
+    extra_args = dict(extra_args)
+    if 'ContentType' not in extra_args:
```

Comment:
We should also add `application/x-iso9660-image`.

Will this still default to `binary/octet-stream` if the type is unrecognized?  If so, we should explicitly fall back to `application/octet-stream`.